routes/web.php

```diff
@@ -1,23 +1,15 @@
 <?php
 
 use App\Http\Controllers\OrganizationController;
-use App\Http\Controllers\SuggestOrganizationController;
-use App\Models\Organization;
-use App\Models\Technology;
+use App\Http\Controllers\SuggestionsController;
 use Illuminate\Support\Facades\Route;
 
 Route::get('/', [OrganizationController::class, 'index'])->name('home');
-Route::get('orgs/{organization}', function (Organization $organization) {
-    return view('organizations.show', ['organization' => $organization]);
-})->name('organizations.show');
-Route::get('suggest', function () {
-    return view('suggestions.create', ['technologies' => Technology::orderBy('name')->get()]);
-})->name('suggestions.create');
-Route::post('suggest', SuggestOrganizationController::class)->name('suggestions.store');
 
-require __DIR__ . '/auth.php';
+Route::resource('organizations', OrganizationController::class)->only('show');
+Route::resource('suggestions', SuggestionsController::class)->only(['create', 'store']);
 
-Route::get('{technology}', [OrganizationController::class, 'indexbyTechnology'])->name('technologies.show');
+require __DIR__ . '/auth.php';
```

Comment:
I still want the URL to be `builtwithlaravel.test/inertia` for technologies, so we'll have to add a route back for that--I know it's not as RESTful, but I don't care, it's what feels good :)

Reply:
Done.